### PR TITLE
Use compound bindings to clean up the bar class computation code.

### PR DESCRIPTION
**Pull request by bicknellr**
Mostly indentation..

### CODE REVIEW

**Comment by abdonrd** [comment]
LGTM if use mixins :+1:

File: paper-toolbar.html [hunk 1]
```diff
@@ -233,150 +233,120 @@
...
       <content select=":not(.middle):not(.bottom)"></content>
     </div>
 
-    <div id="middleBar" class$="[[_computeBarClassName(middleJustify)]]">
+    <div id="middleBar" class$="center horizontal layout toolbar-tools [[_computeBarExtraClasses(middleJustify)]]">
       <content select=".middle"></content>
     </div>
 
-    <div id="bottomBar" class$="[[_computeBarClassName(bottomJustify)]]">
+    <div id="bottomBar" class$="center horizontal layout toolbar-tools [[_computeBarExtraClasses(bottomJustify)]]">
```
Comment (blasten) [inline paper-toolbar.html]: I'd recommend to create a local class and use mixins instead. These classes are coming from paper-styles and they use `/deep/` which is deprecated.

Comment (notwaldorf) [inline paper-toolbar.html]: I actually have a different PR out to fix that: https://github.com/PolymerElements/paper-toolbar/pull/55, so it's not a big deal

**Comment by blasten** [comment]
@bicknellr would you mind rebasing your branch?

**Comment by bicknellr** [comment]
Rebased!

**Comment by blasten** [comment]
merged, thanks!

**Pull request merged**